src/pages/AuthorPage/AboutUsPage.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
this file in a wrong directory please move it to `src/pages/AboutUsPage/AboutUsPage.js`